src/libraries/System.Text.Json/src/System/Text/Json/Nodes/Generics/JsonValue.cs

```diff
@@ -3,8 +3,9 @@
 
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization.Metadata.Generics;
 
-namespace System.Text.Json.Nodes
+namespace System.Text.Json.Nodes.Generics
```

Comment:
Please avoid including large refactor without first discussing with the owners. It can have many unintentional effects.